tests/helpers/test_update_coordinator.py

```diff
@@ -665,8 +665,8 @@ async def test_async_config_entry_first_refresh_no_entry(hass: HomeAssistant) ->
     with pytest.raises(
         RuntimeError,
         match="Detected code that uses `async_config_entry_first_refresh`, "
-        "which is only supported for coordinators with a config entry and will "
-        "stop working in Home Assistant 2025.11. Please report this issue",
+        "which is only supported for coordinators with a config entry. "
+        "Please report this issue",
```

Comment:
`with pytest.raises` => it's already stopped working so we no longer display the breaking version